packages/static-wado-creator/lib/util/adaptProgramOpts.js

```diff
@@ -0,0 +1,37 @@
+module.exports = function adaptProgramOpts(programOpts) {
+  const {
```

Comment:
Use inheritance to start with the defaults value, as there might be more than just this set here, eg:
adaptProgramOpts(programOpts, defaults)
and then extract the new defaults from defaults, and make this an async function so that the default configuration can get loaded here (when present)